.gitignore

```diff
@@ -1,5 +1,6 @@
 /pushgateway
 /.build
+/.idea
```

Comment:
We cannot really accommodate individual IDEs in .gitignore.

Reply:
Got it, I have reverted this part of the PR.